spec/mail/securityconsiderations.mdown

```diff
@@ -10,7 +10,32 @@ Automatic conversion of charsets that allow hidden channels for ASCII text, such
 
 To allow the client to restrict the volume of data it can receive in response to a request, a maximum length may be requested for the data returned for a textual body part. However, truncating the data may change the semantic meaning, for example truncating a URL changes its location. Servers that scan for links to malicious sites should take care to either ensure truncation is not at a semantically significant point, or to rescan the truncated value for malicious content before returning it.
 
-## Email Submission
+## HTML email display
+
+HTML message bodies provide richer formatting for emails but present a number of security challenges especially when embedded in a webmail context in combination with interface HTML. Clients that render HTML email should make careful consideration of the potential risks, including:
+
+* If JavaScript is allowed to execute it can rewrite email to change its
```

Comment:
I'd rewrite this to explicitly suggest that the client must disallow JavaScript. This can be achieved with `iframe[sandbox]`.